### ISSUE TITLE
What do "Clover Audio ID Injection" and "Use Audio ID: x" do?

### ISSUE THREAD

**Issue by dudeofawesome** [toleda/audio_CloverALC#49]
What do "Clover Audio ID Injection" and "Use Audio ID: x" do?

The README.md is somewhat lacking in description of what the Audio ID Injection will do.

It appears that the Audio ID that you should pick depends on what codec your motherboard supports, but also on some other things such as the number of audio ports you have and what devices you want to support HDMI audio.

> 2. Supported Audio IDs
>   - Audio ID: 1 supports 269, 283, 885, 887, 888, 889, 892, 898, 1150
>     
>     Realtek ALC audio (default, 1/2/3/5/6 motherboard audio ports)
>   - Audio ID: 2 supports 887, 888, 889, 892, 898, 1150
>     
>     Realtek ALC/5.1 surround sound (3 motherboard audio ports)
>   - Audio ID: 3 supports 887, 888, 889, 892, 898, 1150
>     
>     HD3000/HD4000 HDMI audio with Realtek ALC audio

There seems to be some more info in  [https://github.com/toleda/audio_ALCInjection/blob/master/README.txt](https://github.com/toleda/audio_ALCInjection/blob/master/README.txt), but this also seems to contradict some of the info.

> Audio_ID: 1 supports 5 and 6 port ALC8xx onboard and/or HD5K/AMD/Nvidia HDMI audio  
> Audio_ID: 2 supports 3 port ALC8xx onboard and/or HD5K/AMD/Nvidia HDMI audio
> Audio_ID: 3 supports 3, 5 and 6 port ALC8xx onboard HD4K/HD3K HDMI audio with or without AMD/Nvidia HDMI audio
> Audio_IDs: 1 and 2 support analog 5.1 surround sound, 3 does not
> Audio_IDs: 1, 2 and 3 require HDMI audio dsdt edits for HDMI audio
> Audio_ID: 3, not supported with ALC1150.

[I've attempted to update the README.md](https://github.com/dudeofawesome/audio_CloverALC/blob/master/README.md) to better refelect my understanding of these documents. Does this look correct?

Thanks for all your work on hackintosh audio!

**Comment by toleda** [comment]
Thanks.  The lower case Roman numerals don't work very well, i.e.,  Audio ID: i.

cloverALC is specifically for onboard audio.  For HD3000/HD4000 HDMI audio, Audio IDs: 1 and 2 do not work; only Audio ID: 3 works.  All other HDMI audio configurations work with Audio ID; 1 or 2.  

More info, see https://github.com/toleda/audio_CloverHDMI.

**Comment by dudeofawesome** [comment]
Shoot. You're right. I was looking at the markdown version in Atom, and it was displaying numbers, not roman numerals. I'm not sure what's up with that. I'll update that to make it work.

I'll also change the wording on the HDMI audio parts to better reflect that cloverALC doesn't actually enable them.

**Comment by dudeofawesome** [comment]
Do audio IDs 1 and 2, but not 3, work with HD4600?

**Comment by toleda** [comment]
No, HD4600 HDMI audio does not use dsdt/HDEF. Audio ID: 3 works, however, Orange is not supported.  No reason to use Audio ID: 3 with HD4600 HDMI audio.
